app/javascript/dashboard/routes/dashboard/settings/macros/constants.js

```diff
@@ -14,6 +14,11 @@ export const MACRO_ACTION_TYPES = [
     label: 'Add a label',
     inputType: 'multi_select',
   },
+  {
+    key: 'remove_label',
+    label: 'Remove a label',
```

Comment:
Maybe we can rename this to "Remove Labels" instead of "Remove a label", since it's letting me remove multiple labels